cmd/ddev/cmd/start.go

```diff
@@ -28,28 +30,34 @@ provide a local development environment.`,
 		dockerutil.EnsureDdevNetwork()
 	},
 	Run: func(cmd *cobra.Command, args []string) {
-		appStart()
-	},
-}
+		apps, err := getRequestedApps(args, startAll)
+		if err != nil {
+			util.Failed("Unable to start project(s): %v", err)
+		}
 
-// appStart is a convenience function to encapsulate startup functionality
-func appStart() {
-	app, err := ddevapp.GetActiveApp("")
-	if err != nil {
-		util.Failed("Failed to start project: %v", err)
-	}
+		if len(apps) == 0 {
+			output.UserOut.Printf("There are no stopped projects to start.")
```

Comment:
Start doesn't just start stopped projects. It also re-runs docker-compose up on running projects. I use it that way a lot. It's just kind of ambiguous.